src/workerd/api/streams/internal.c++

```diff
@@ -1515,8 +1522,8 @@ jsg::Promise<void> WritableStreamInternalController::writeLoopAfterFrontOutputLo
   static constexpr auto inspectQueue = [](auto& queue, kj::StringPtr name) {
     if (queue.size() > 1) {
       kj::Vector<kj::String> events;
-      for (auto& event: queue) {
-        KJ_SWITCH_ONEOF(event.event) {
+      queue.forEach([&](auto& next) {
```

Comment:
[For a future PR]: This function is 300 lines long; that is way too much IMO.

Reply:
What?! It's just a wee baby function! 👶🏻